werkzeug/utils.py

```diff
@@ -423,13 +423,7 @@ def import_string(import_name, silent=False):
             return sys.modules[import_name]
 
         module_name, obj_name = import_name.rsplit('.', 1)
-        try:
-            module = __import__(module_name, None, None, [obj_name])
-        except ImportError:
-            # support importing modules not yet set up by the parent module
-            # (or package for that matter)
-            module = import_string(module_name)
```

Comment:
I can't find a way to break your code either. The original code dates back to https://github.com/mitsuhiko/werkzeug/commit/9aa53b8a8a7203bcbaf62d2ed1a12c8d7e54d3ce, but there's insufficient explanation there.